specutils/spectra/spectrum1d.py

```diff
@@ -7,7 +7,7 @@
 from astropy.nddata import NDDataRef
 from astropy.utils.decorators import lazyproperty
 from .spectrum_mixin import OneDSpectrumMixin
-from .spectral_coordinate import SpectralCoord
+from ..extern.spectralcoord import SpectralCoord
```

Comment:
```suggestion
```

This can be removed since we don't use `SpectralCoord` in `Spectrum1D` anymore (it's used through `SpectralAxis`).

Reply:
Fixed